apps/pie-storybook/stories/pie-cookie-banner.stories.ts

```diff
@@ -32,15 +32,15 @@ const cookieBannerStoryMeta: CookieBannerStoryMeta = {
             control: 'boolean',
         },
         country: {
-            options: Object.values(Country),
+            options: [...Object.values(Country), 'ru', 'pt', 'ES', 'invalid'], // Expanded to allow for unsupported value tests
```

Comment:
Are the unsupported values displayed at all times in the storybook? (as it looks like these are being added for tests to run against it?)

Just wondering if this could potentially be confusing (or if there's a way of only adding these if the tests are running against storybook)

Reply:
This is probably one for @siggerzz  and @JoshuaNg2332 to look at later. I think there may be plans to separate it out a bit, but can catch up about this separately 👍